server/src/controllers/Account.ts

```diff
@@ -18,29 +21,58 @@ import * as Auth from '../lib/auth';
 @JsonController()
 export class AccountController {
   @Get('/accounts/')
+  @OpenAPI({
+    summary: 'Return a list of accounts',
+    description: 'List all accounts',
+  })
   public async getAll() {
     return Account.find();
   }
 
   @Post('/accounts/')
+  @OpenAPI({
+    summary: 'Add one account',
+    description: 'Add an account to the Account table',
+  })
+  @ResponseSchema(Account, {
+    contentType: 'text/csv',
+    description: 'A list of created account objects',
```

Comment:
A list of created Accounts (don't use objects because it is not actually objects to the user).